selfdrive/controls/controlsd.py

```diff
@@ -624,10 +627,13 @@ def publish_logs(self, CS, start_time, CC, lac_log):
     if current_alert:
       hudControl.visualAlert = current_alert.visual_alert
 
+    frame_id = CC.frameId
     if not self.read_only and self.initialized:
       # send car controls over can
       self.last_actuators, can_sends = self.CI.apply(CC)
-      self.pm.send('sendcan', can_list_to_can_capnp(can_sends, msgtype='sendcan', valid=CS.canValid))
+      sendcan_bytes = can_list_to_can_capnp(can_sends, msgtype='sendcan', valid=CS.canValid)
+      cloudlog.timestampExtra("translation", {"from": log.Event.from_bytes(sendcan_bytes).logMonoTime, "to": frame_id})
```

Comment:
why is this done here? this LUT can be recovered from the rest of the logs and implementation details like this shouldn't leak over here

Reply:
How do you bind the logMono to the frame Id in the logs? You would get a planner message with frame Id and later a sendcan with logMono. But how do we know that these correspond to the same?

If the answer is that we can assume that the message after a planner message is a controls message corresponding to the same frame, why do we track the frame id at all? Cant we just assume the repeated sequence camera-model-planner-controls-board